client/my-sites/patterns/components/pattern-gallery/client.tsx

```diff
@@ -109,12 +118,21 @@ export const PatternGalleryClient: PatternGalleryFC = ( props ) => {
 		patternTypeFilter,
 	} = props;
 
+	const translate = useTranslate();
+	const [ patternDisplayCount, setPatternDisplayCount ] = useState( PATTERNS_PER_PAGE_COUNT );
 	const isLoggedIn = useSelector( isUserLoggedIn );
 	const patternIdsByCategory = {
 		first: patterns.map( ( { ID } ) => `${ ID }` ) ?? [],
 	};
 	const isPageLayouts = patternTypeFilter === PatternTypeFilter.PAGES;
 
+	const patternsToDisplay = patterns.slice( 0, patternDisplayCount );
+	const isDisplayingPaginationButton = patternsToDisplay.length < patterns.length;
```

Comment:
Small nit/question:

It feels like this declaration checks to see if we _should_ display a pagination button (are there more patterns than we plan to display initially) rather than confirming that we _are_ displaying it (i.e. if we were confirming its presence in the DOM). Would it be preferable to name this variable `shouldDisplayPaginationButton`?

Reply:
Good call 👍